specification/cognitiveservices/data-plane/CustomVision/Training/stable/v3.0/Training.json

```diff
@@ -0,0 +1,4494 @@
+            "collectionFormat": "csv",
+            "x-nullable": true
+          },
+          {
+            "$ref": "#/parameters/ApiKey"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CustomVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful DeleteImageRegions request": {
+            "$ref": "./examples/DeleteImageRegions.json"
+          }
+        }
+      }
+    },
+    "/projects/{projectId}/images/tagged": {
+      "get": {
+        "tags": [
+          "ImageApi"
+        ],
+        "summary": "Get tagged images for a given project iteration.",
+        "description": "This API supports batching and range selection. By default it will only return first 50 images matching images.\r\nUse the {take} and {skip} parameters to control how many images to return in a given batch.\r\nThe filtering is on an and/or relationship. For example, if the provided tag ids are for the \"Dog\" and\r\n\"Cat\" tags, then only images tagged with Dog and/or Cat will be returned",
+        "operationId": "GetTaggedImages",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "name": "projectId",
+            "in": "path",
+            "description": "The project id.",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-nullable": false
+          },
+          {
+            "name": "iterationId",
+            "in": "query",
+            "description": "The iteration id. Defaults to workspace.",
+            "required": false,
+            "type": "string",
+            "format": "uuid",
+            "x-nullable": true
+          },
+          {
+            "name": "tagIds",
+            "in": "query",
+            "description": "A list of tags ids to filter the images. Defaults to all tagged images when null. Limited to 20.",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string",
+              "x-nullable": false
+            },
+            "collectionFormat": "csv",
+            "x-nullable": true
+          },
+          {
+            "name": "orderBy",
+            "in": "query",
+            "description": "The ordering. Defaults to newest.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "Newest",
+              "Oldest"
+            ],
+            "x-nullable": false
+          },
+          {
+            "name": "take",
+            "in": "query",
+            "description": "Maximum number of images to return. Defaults to 50, limited to 256.",
```

Comment:
You can use [`maximum` and `minimum`](https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-5.1.2) to specify max and min values. Which I would recommend for input-only types (but not for types returned by the service as this can become too restrictive/cause versioning problems in the future)